docs/extended-cpts.md

```diff
@@ -0,0 +1,312 @@
+# Extended Custom Post Types
+
+For the full documentation, refer to the [Extended CPTs wiki](https://github.com/johnbillion/extended-cpts/wiki).
+
+## Registering Post Types
+
+The `register_extended_post_type()` function is the core of Extended CPTs. It's a wrapper for WordPress' own `register_post_type()` function, which means that any parameters which are accepted by `register_post_type()` are accepted.
+
+The function's signature looks like this:
+
+```php
+function register_extended_post_type( string $post_type, array $args = [], array $names = [] ) : Extended_CPT
```

Comment:
Note to self: need to standardise how we do these function signatures and document that somewhere.

Think other examples like this just omit `function` from the start eg

```suggestion
register_extended_post_type( string $post_type, array $args = [], array $names = [] ) : Extended_CPT
```